Sources/Public/Animation/LottieView.swift

```diff
@@ -299,12 +317,16 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
   /// whenever the `binding` value is updated.
   ///
   /// - Note: This function requires a valid `loadAnimation` closure provided during view initialization,
-  ///         otherwise the `loadAnimationTrigger` will have no effect.
-  /// - Note: The existing animation will be removed before calling `loadAnimation`,
-  ///         which will cause the `Placeholder` to be displayed until the new animation finishes loading.
-  public func loadAnimationTrigger<Value: Hashable>(_ binding: Binding<Value>) -> Self {
+  ///         otherwise `reloadAnimationTrigger` will have no effect.
+  /// - Parameters:
+  ///   - binding: The binding that triggers the reloading when its value changes.
+  ///   - showPlaceholder: When `true`, the current animation will be removed before invoking `loadAnimation`,
+  ///     displaying the `Placeholder` until the new animation loads.
+  ///     When `false`, the previous animation remains visible while the new one loads.
+  public func reloadAnimationTrigger<Value: Equatable>(_ binding: Binding<Value>, showPlaceholder: Bool = true) -> Self {
     var copy = self
-    copy.loadAnimationTrigger = binding.map(transform: AnyHashable.init)
+    copy.reloadAnimationTrigger = binding.map { $0 as any Equatable }
```

Comment:
We cant use `any Equatable` due to backwards compatibility requirements (CI is failing), so I think we need to switch back to requiring `Hashable` so we can use `AnyHashable` here. Fortunately most `Equatable` types are also `Hashable`.